src/components/Layout/Layout.tsx

```diff
@@ -232,20 +264,17 @@ export const Layout = ({
                 </View>
               </Flex>
               <View
-                className={`layout-sidebar${
-                  menuOpen ? ' layout-sidebar--expanded' : ''
-                }`}
+                className={`layout-sidebar${menuOpen ? ' layout-sidebar--expanded' : ''
+                  }`}
               >
                 <View
-                  className={`layout-sidebar__backdrop${
-                    menuOpen ? ' layout-sidebar__backdrop--expanded' : ''
-                  }`}
+                  className={`layout-sidebar__backdrop${menuOpen ? ' layout-sidebar__backdrop--expanded' : ''
+                    }`}
                   onClick={() => toggleMenuOpen(false)}
                 ></View>
                 <View
-                  className={`layout-sidebar__inner${
-                    menuOpen ? ' layout-sidebar__inner--expanded' : ''
-                  }`}
+                  className={`layout-sidebar__inner${menuOpen ? ' layout-sidebar__inner--expanded' : ''
+                    }`}
```

Comment:
Are these just prettier changes?

Reply:
they were just prettier changes but I've changed these classes to use the `classnames` library instead because its cleaner